packages/docs/getting-started/running-attestation-service.md

```diff
@@ -56,39 +57,19 @@ If you prefer using Twilio, this is list of the variables to use:
 
 ## Accounts Configuration
 
-First we need to create an account for getting the attestation key needed to sign the attestations. Run:
+First, the validator account needs to authorize the attestation signer. With `celocli` connected to a node that has the attestation signer unlocked, let's create a proof-of-posession:
 
 ```bash
-celocli account:new
+celocli account:proof-of-possession --signer $ATTESTATION_SIGNER_ADDRESS --account $CELO_VALIDATOR_ADDRESS
```

Comment:
I don't see `CELO_VALIDATOR_ADDRESS` listed anywhere in the env variables you're expecting